graph/src/env/mod.rs

```diff
@@ -391,6 +401,11 @@ struct Inner {
     enable_gas_metrics: EnvVarBoolean,
     #[envconfig(from = "GRAPH_EXPERIMENTAL_TIMESERIES")]
     enable_timeseries: Option<bool>,
+    #[envconfig(from = "GRAPH_HISTORY_BLOCKS_OVERRIDE")]
+    history_blocks_override: Option<BlockNumber>,
+    // TODO: Find a good default value
```

Comment:
We need to decide what a good default value for min is

Reply:
I'd propose `2*ETHEREUM_REORG_THRESHOLD`, short but safe. @lutter ideas?

Reply:
Yeah, anything bigger than reorg threshold should be safe. Some small multiple sounds good.

Reply:
Made that change to use 2*reorg threshold